.github/workflows/execution-plan-main.yml

```diff
@@ -13,6 +13,12 @@ jobs:
       trigger: main
     secrets: inherit
 
+  integration-tests:
```

Comment:
add this to execution-plan-pr.yaml, otherwise it won't run until you merge this PR

Reply:
I addressed this in the comment above